.gitignore

```diff
@@ -1,5 +1,8 @@
 /node_modules/
```

Comment:
looks like this version colors the completion list in blue for some reason when running watch. This is not the case for the older version (and should not be the case for the runtime as well).  Any idea why that is?
![image](https://user-images.githubusercontent.com/199509/78635394-f752a600-785a-11ea-8ef6-d5c133ab393d.png)

Reply:
This is really odd.
I see function class in my branch has this color:
![image](https://user-images.githubusercontent.com/13627367/79462118-c5320880-7fff-11ea-94ad-9ca6c5ff6a8d.png)

God knows why. Investigating..

Reply:
It comes from this style element, I have no idea how it gets added to index.html, maybe something to do with not opening the index.html as a static site, and then live-server/http-server pushes this default style element. This happens only when you test the package of course, not consume it.
I added a style element to index.html to override this.
![image](https://user-images.githubusercontent.com/13627367/79467710-c286e180-8006-11ea-8b76-fb94d72ce9cf.png)